TombEngine/Objects/TR5/Entity/tr5_larson.cpp

```diff
@@ -73,36 +77,28 @@ namespace TEN::Entities::Creatures::TR5
 		short joint2 = 0;
 
 		// TODO: When Larson's HP is below 40, he runs away in Streets of Rome. Keeping block commented for reference.
-		/*if (item->HitPoints <= TR5_LARSON_MIN_HP && !(item->flags & IFLAG_INVISIBLE))
+		if (item->HitPoints <= TR5_LARSON_MIN_HP && !(item->Flags & IFLAG_INVISIBLE))
 		{
 			item->HitPoints = TR5_LARSON_MIN_HP;
-			creature->flags++;
-		}*/
+			creature->Flags++;
+		}
 
 		if (creature->MuzzleFlash[0].Delay != 0)
 			creature->MuzzleFlash[0].Delay--;
-
 		if (creature->MuzzleFlash[1].Delay != 0)
 			creature->MuzzleFlash[1].Delay--;
 
 		if (item->TriggerFlags)
 		{
 			if (CurrentLevel == 2)
 			{
-				item->Animation.IsAirborne = false;
```

Comment:
Why all these fields are replaced by `item->AIBits = AMBUSH;`? Same for cases below.

Reply:
I ported TOMB5 code but it's still wont work because it's missing the last comment at the end.